forms-flow-theme/webpack.config.js

```diff
@@ -1,39 +1,51 @@
-const { merge } = require("webpack-merge");
-const singleSpaDefaults = require("webpack-config-single-spa");
+const path = require("path");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
+module.exports = (env, argv) => {
+  const isProduction = argv.mode === "production";
 
-module.exports = (webpackConfigEnv, argv) => {
-  const defaultConfig = singleSpaDefaults({
-    orgName: "formsflow",
-    projectName: "theme",
-    webpackConfigEnv,
-    argv,
-  });
-
-  return merge(defaultConfig, {
-    // modify the webpack config however you'd like to by adding to this object
+  return {
+    entry: "./src/scss/index.scss",
+    output: {
+      path: path.resolve(__dirname, "dist"),
+    },
     devServer: {
       headers: {
         "Access-Control-Allow-Origin": "*",
       },
-      port: 3008
-    },
-    output:{
-      filename:"forms-flow-theme.js"
+      port: 3008,
+      watchFiles: {
+        paths: ['./src/scss/'],
+        options: {
+          ignored: ['node_modules/**'] // Example of files to ignore
+        }
+      }
+  
     },
     module: {
       rules: [
         {
           test: /\.s[ac]ss$/i,
           use: [
-            // Creates `style` nodes from JS strings
-            "style-loader",
-            // Translates CSS into CommonJS
+            MiniCssExtractPlugin.loader,
             "css-loader",
-            // Compiles Sass to CSS
             "sass-loader",
           ],
         },
       ],
     },
-  });
+    plugins: [
+      new MiniCssExtractPlugin({
+        filename: "forms-flow-theme.min.css",
+        chunkFilename: isProduction ? "[id].css" : "[id].css",
```

Comment:
isn't this ternary using same resolution, i.e  `[id].css` in both cases ?

Reply:
Will create a ticket for this and then do the required changes , then move this PR. For the time being keeping this PR as draft

Reply:
https://aottech.atlassian.net/browse/FWF-3267